OracleDatabase/RAC/OracleRealApplicationClusters/dockerfiles/18.3.0/resetOSPassword.sh

```diff
@@ -58,8 +60,12 @@ echo  $PASSWORD
 }
```

Comment:
echo $PASSWORD should be commented out or only enabled when debugging

Reply:
Thanks. I fixed it.